doc/classes/PhysicsServer3DExtension.xml

```diff
@@ -760,6 +760,11 @@
 			<description>
 			</description>
 		</method>
+		<method name="_is_active" qualifiers="virtual const">
+			<return type="bool" />
+			<description>
```

Comment:
```suggestion
			<description>
				Overridable version of [method PhysicsServer3D.is_active].
```

Reply:
Is there some way that I can amend the commit with this change? The docs say that PRs should usually use just one commit...

Reply:
You can just use `git commit --amend` and then push with `git push -f`